### ISSUE TITLE
Uninitialized values in QHexEdit constructor

### ISSUE THREAD

**Issue by akryukov** [Simsys/qhexedit2#32]
Since all setters for main class fields are closely interconnected, calling anyone of them while some other fields haven't yet been initialized leads to conditionals relying on uninitialized values (you can check this with valgrind).

To circumvent this I had just to add to the QHexEdit constructor the following lines:
    _addressArea = true;
    _addressWidth = 4;
    _asciiArea = true;
    _overwriteMode = true;
    _highlighting = true;
    _readOnly = false;
    _cursorPosition = 0;
    _lastEventSize = 0;

Not sure if it is the correct solution, however.

**Comment by Simsys** [comment]
Can you provide a pull request?

Thanks!

**Comment by justinclift** [comment]
@akryukov ping?

**Comment by revolter** [comment]
@Simsys, Is this (partially) fixed by @6271425e? It looks like these:

```
    setAddressWidth(4);
    setAddressArea(true);
    setAsciiArea(true);
    setOverwriteMode(true);
    setHighlighting(true);
    setReadOnly(false);
```

are similar to what @akryukov asked for.

**Comment by Simsys** [comment]
fixed, see 4e8821c288ad5820c8fa5bd713940692cee883d8
